api/middleware.go

```diff
@@ -279,8 +279,8 @@ func RuntimeFromURLMiddleware(baseURL string) func(next http.Handler) http.Handl
 				runtime = common.RuntimeSapphire
 			case strings.HasPrefix(path, "/cipher/"):
 				runtime = common.RuntimeCipher
-			case strings.HasPrefix(path, "/pontusx/"):
-				runtime = common.RuntimePontusx
+			case strings.HasPrefix(path, "/pontusxtest/"):
+				runtime = common.RuntimePontusxTest
```

Comment:
Note: Explorer folks will have to update this since pontusx data on testnet will now be served under the `pontusxtest` prefix. If/when pontusx on mainnet is enabled, this section will need to be updated as well.